requirements.txt

```diff
@@ -35,12 +34,12 @@ regex==2021.4.4
 requests==2.25.1
 resampy==0.2.2
 scikit-learn==0.24.2
-scipy==1.6.2
+scipy==1.5.4
 six==1.15.0
 SoundFile==0.10.3.post1
 threadpoolctl==2.1.0
-torch==1.8.1
-torchaudio==0.8.1
+torch==1.9.0
```

Comment:
Like I said before, you will need to rebase on the latest version of `main` so that these changes don't show up here.